cirq-core/cirq/experiments/qubit_characterizations.py

```diff
@@ -464,6 +516,18 @@ def _measurement(two_qubit_circuit: circuits.Circuit) -> np.ndarray:
     return TomographyResult(rho)
 
 
+def _create_parallel_rb_circuit(
+    qubits: Iterator['cirq.Qid'], num_cliffords: int, c1: list, clifford_mats: np.ndarray
+) -> 'cirq.Circuit':
+    circuits_to_zip = [
+        _random_single_q_clifford(qubit, num_cliffords, c1, clifford_mats) for qubit in qubits
+    ]
+    circuit = circuits.Circuit.zip(*circuits_to_zip)
+    measure_moment = circuits.Moment(ops.measure_each(*qubits))
+    circuit_with_meas = circuits.Circuit.from_moments(*circuit.moments, measure_moment)
+    return circuit_with_meas
```

Comment:
nit: can inline measure_each call in the from_moments, and return the final circuit without a local var. Also, iterating over a circuit yields the moments, so don't need to call `.moments` directly:
```suggestion
    return circuits.Circuit.from_moments(*circuit, ops.measure_each(*qubits))
```